### ISSUE TITLE
Not Resolving Android Dependencies

### ISSUE THREAD

**Issue by sogonleong** [playgameservices/play-games-plugin-for-unity#2576]
0.9.64 does not resolve android dependencies. I thought why it wasn't working, then noticed there are no aar files downloaded under /Plugins/Android

**Comment by Noah1911** [comment]
https://github.com/googlesamples/unity-jar-resolver try to download newer Jar Resolver and see if it works.

**Comment by sogonleong** [comment]
Using the latest Unity Jar Resolver seems to solve the issue that my app wasn't loading into Google Play Game Services but there's still no aar files under /Plugins/Android.

Is this the new process? That it will auto download and compile when building?

**Comment by karlseven** [comment]
Yes i have the same problem too, Libraries are not being resolved unlike before

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
